src/plot/ROOTMultiPlot.cpp

```diff
@@ -77,7 +77,10 @@ ROOTMultiPlot::Construct(){
   for(size_t i = 0; i < fNames.size(); i++){
     fLegend.AddEntry(&fHists.at(i), fNames.at(i).c_str());
     fHists[i].SetLineColor(fColorScheme[i %fNcolors]);
-    fHists.at(i).Draw("same");
+    if(i!=0)
```

Comment:
I'm pretty sure this is already the default behaviour, Draw("same") behaves like Draw() if nothing has been drawn before